### PR TITLE
#27 added logout button in drawer

### PR DESCRIPTION
**Pull request by lea88pu**
Please review

### CODE REVIEW

**Review by goranesine — APPROVED** [review]

File: lib/mvos/ui/widgets/drawer.dart [hunk 1]
```diff
@@ -12,23 +14,60 @@ class CustomDrawer extends StatelessWidget {
...
+                            child: CircularProgressIndicator(
+                              backgroundColor: Colors.pink,
+                            ),
+                          );
+                      });
+                  if (data2.loggedIn == true) {
+                    o.logOut().then((_) {
+                      Navigator.pushReplacementNamed(
+                          context, RouteName.logInPage);
+                      print('ruta');
```
Comment (matejthetree) [inline lib/mvos/ui/widgets/drawer.dart]: Print vani

Comment (matejthetree) [reply lib/mvos/ui/widgets/drawer.dart]: Makni

File: lib/mvos/ui/widgets/drawer.dart [hunk 2]
```diff
@@ -12,23 +14,60 @@ class CustomDrawer extends StatelessWidget {
...
+                  showDialog(
+                      context: context,
+                      builder: (BuildContext context) {
+                        return Center(
+                            child: CircularProgressIndicator(
+                              backgroundColor: Colors.pink,
+                            ),
+                          );
+                      });
+                  if (data2.loggedIn == true) {
```
Comment (matejthetree) [inline lib/mvos/ui/widgets/drawer.dart]: Ovaj if treba ici na consumer.
```
Data2.loggedIn?Consumer(...):Contianer()
```
Ako nije logged in ide prazan kontner.

**Review by matejthetree — CHANGES_REQUESTED** [review]
Ostalo je super

File: lib/mvos/ui/widgets/loginPage_form.dart [hunk 3]
```diff
@@ -22,7 +22,7 @@ class _LoginFormState extends State<LoginForm> {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.all(8.0),
+      padding: const EdgeInsets.only(left:8.0,right: 8.0),
```
Comment (matejthetree) [inline lib/mvos/ui/widgets/loginPage_form.dart]: you didnt format correctly

**Review by matejthetree — CHANGES_REQUESTED** [review]

**Review by djkowa — COMMENTED** [review]
reviewed

**Review by matejthetree — APPROVED** [review]

**Pull request merged**